core/designsystem/src/main/kotlin/com/dhc/designsystem/bottomsheet/DhcBottomSheet.kt

```diff
@@ -0,0 +1,93 @@
+package com.dhc.designsystem.bottomsheet
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.ColumnScope
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.ModalBottomSheet
+import androidx.compose.material3.SheetValue
+import androidx.compose.material3.Surface
+import androidx.compose.material3.rememberModalBottomSheetState
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.dhc.designsystem.LocalDhcColors
+import com.dhc.designsystem.R
+import com.dhc.designsystem.SurfaceColor
+import kotlinx.coroutines.launch
+
+@Composable
+@OptIn(ExperimentalMaterial3Api::class)
+fun DhcModalBottomSheet(
+    isCloseButtonEnabled: Boolean,
+    modifier: Modifier = Modifier,
+    onDismissRequest: () -> Unit,
+    containerColor: Color = SurfaceColor.neutral700,
+    content: @Composable ColumnScope.() -> Unit,
+) {
```

Comment:
@ddyeon 
nit) 파라미터 순서 변경하면 더 좋을듯 !
```suggestion
fun DhcModalBottomSheet(
    isCloseButtonEnabled: Boolean,
    onDismissRequest: () -> Unit,
    modifier: Modifier = Modifier,
    containerColor: Color = SurfaceColor.neutral700,
    content: @Composable ColumnScope.() -> Unit,
) {
```